types/src/front/assistant/actions/browse.ts

```diff
@@ -0,0 +1,40 @@
+import * as t from "io-ts";
+
+import { ModelId } from "../../../shared/model_id";
+import { BaseAction } from "../../lib/api/assistant/actions";
+
+export type BrowseConfigurationType = {
+  id: ModelId;
+  sId: string;
+  type: "browse_configuration";
+  name: string | null;
+  description: string | null;
+};
+
+export const BrowseResultSchema = t.type({
+  content: t.string,
```

Comment:
what about including the browsed URLs in the schema? so that it can be known which browsed data is which url definitely (without having to rely on mapping to the urls string[]) -- for us in code (e.g. if we move the array around) and for the model when rendering the action result

Reply:
Actually already returned by the dust app, just forgot to add it to the schema :-)